hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -581,7 +631,9 @@ static QString* psh_convert(struct PixelShader *ps)
     qstring_append(vars, "vec4 pD1 = vtx.D1 / vtx.inv_w;\n");
     qstring_append(vars, "vec4 pB0 = vtx.B0 / vtx.inv_w;\n");
     qstring_append(vars, "vec4 pB1 = vtx.B1 / vtx.inv_w;\n");
-    qstring_append(vars, "vec4 pFog = vec4(fogColor.rgb, clamp(vtx.Fog / vtx.inv_w, 0.0, 1.0));\n");
+    qstring_append(vars,
+                   "vec4 pFog = vec4(fogColor.rgb, clamp(vtx.Fog / vtx.inv_w, "
+                   "0.0, 1.0));\n");
```

Comment:
I'd undo this regardless of 80 col limit, simply because it's in the same block with many similar lines, so there's no reason why this should stand out.